### PR TITLE
New tutorial proposal - Get started with Corda

### PR DESCRIPTION
**Pull request by Sphar-io**
## Done
* Added complete draft of new tutorial module 

* This tutorial covers how to setup and run a simple distributed Corda application

### CODE REVIEW

File: tutorials/server/get-started-with-corda/get-started-with-corda.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+```
+* Navigate into the cloned directory.
+
+**We can now run this CorDapp from the terminal**
+
+
+## Running from terminal
+Duration: 5:00
+
+To begin the setup process, start the deploy nodes Gradle task
```
Comment (caldav) [inline tutorials/server/get-started-with-corda/get-started-with-corda.md]: Suggestion: "the `deployNodes` Gradle task"

File: tutorials/server/get-started-with-corda/get-started-with-corda.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+If this command fails with error message `Exception in thread "main" java.io.IOException: Cannot run program "xterm"` you will need to install the xterm terminal. This can be done using apt, with the command 
+```
+sudo apt install xterm
+```
+When finished, rerun the `runnodes` command. 
+
+Wait while seven additional terminal windows open. The start up process is finished when all the terminal windows display either “Webserver started up in XX.X sec” or “Node for “PartyX” started up and registered in XX.XX sec”. 
+
+Test that the CorDapp is running correctly by visiting PartyA's web frontend at <http://localhost:10009/web/example/>
+* From this frontend you can create IOUs and send them to other parties.
```
Comment (caldav) [inline tutorials/server/get-started-with-corda/get-started-with-corda.md]: A screenshot of the web frontend would be a nice addition here

File: tutorials/server/get-started-with-corda/get-started-with-corda.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+
+Under `Project SDK` click `New`, then `JDK`. Chose the location of the Oracle JDK installed in the first step. The default installation location is `/usr/lib/jvm/java-8-oracle`. Click `OK`.
+
+Back in the project structure menu, click `Apply`. Select `Modules` from the left hand navigation pane. 
+Click the green plus button and select `Import Module`. The CorDapp installation location should be selected by default. If not, navigate to it and click `OK`. 
+Select `Import module from external module`, choosing `Gradle` and clicking `Next`. 
+Click `Finish`, leaving default settings. 
+
+Wait while Gradle builds and indexes the project.
```
Comment (caldav) [inline tutorials/server/get-started-with-corda/get-started-with-corda.md]: A screenshot of the resulting state in IntelliJ would be a nice addition

File: tutorials/server/get-started-with-corda/get-started-with-corda.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+
+---
+
+# Get started with Corda
+
+## Overview
+Duration: 1:00
+
+Corda is a blockchain platform built for business. Corda removes costly friction in business transactions by enabling businesses to transact directly. Using smart contract and blockchain technology, Corda allows existing business networks to reduce transaction and record-keeping costs and to streamline business operations. 
+
```
Comment (caldav) [inline tutorials/server/get-started-with-corda/get-started-with-corda.md]: What about the Corda logo here? (see https://tutorials.ubuntu.com/tutorial/continuous-snap-delivery-from-travis-ci#0 for example)

File: tutorials/server/get-started-with-corda/get-started-with-corda.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+### Next steps
+The best way to check that everything is working fine is by taking a deeper look at the example CorDapp.
+
+Next, you should read through the self-guided [Corda Key Concepts course](https://r3.lessonly.com/path/5150-corda-key-concepts/login?view=signup) to understand how Corda works.
+
+By then, you’ll be ready to start writing your own CorDapps. Learn how to do this in the [Hello, World tutorial](https://docs.corda.net/hello-world-introduction.html). You may want to refer to the [API documentation](https://docs.corda.net/api/kotlin/corda/index.html), [the flow cookbook](https://docs.corda.net/flow-cookbook.html) and [the samples](https://www.corda.net/samples/) along the way.
+
+### Support
+If you encounter any issues, please see [the Troubleshooting page](https://docs.corda.net/troubleshooting.html), or ask on [Stack Overflow](https://stackoverflow.com/questions/tagged/corda) or via [our Slack channels](https://slack.corda.net).
```
Comment (caldav) [inline tutorials/server/get-started-with-corda/get-started-with-corda.md]: "our Slack channels" -> "the Corda slack channels"

**Comment by Sphar-io** [comment]
Thanks so much for the review @caldav , I've made the 4 changes you suggested.

**Pull request merged**